src/NLog/Targets/ConsoleTargetHelper.cs

```diff
@@ -41,11 +41,15 @@ internal static class ConsoleTargetHelper
     {
         public static bool IsConsoleAvailable(out string reason)
         {
+            reason = string.Empty;
 #if !SILVERLIGHT && !__IOS__ && !__ANDROID__ && !MONO
             try
             {
                 if (!Environment.UserInteractive)
                 {
+                    if (Internal.PlatformDetector.IsMono && Console.In is StreamReader)
```

Comment:
Ps

> 	
I found a way to figure out whether there is a console/terminal available or not under mono on linux: 'Mono.Unix.Native.Syscall.isatty(0)'.

http://stackoverflow.com/questions/9065065/how-to-detect-if-im-running-in-mono-service